docs/intro/ff-ui/canvas.md

```diff
@@ -0,0 +1,144 @@
+
+It allows zoom level adjustments and previews in light or dark mode. It also includes features for multi-language preview, adding App Bar and Nav Bar, simulating larger font visualization, and more.
+
+![canvas area](imgs/canvas-new.avif)
+
+## 1. Preview Screen
+
+This is the canvas of the device screen where you can build the user interface. You can customize the screen by adding widgets using drag and drop from the [Widget Palette](#) and by applying properties present in the [Properties Panel](#).
+
+## 2. Show or hide Navigation Menu
+
+From here, you can open or close the [Navigation Menu](#).
+
+## 3. Zoom Controls
+
+There's a zoom in ( + ) and a zoom out ( - ) button to control the zoom level of the canvas. While working on complex user interface designs, this comes in handy to give you a better preview of the widgets by zooming in to a particular area and then zooming out to have an overview.
+
+## 4. AI Generated Page
+
+From here, you can [create a page using AI](#).
+
+## 5. Add App Bar
+
+From here, you can add an [App Bar](#) to your page. Clicking this button opens a popup displaying different [App Bar styles](#) for you to choose from. Just select any App Bar style from the list and it will appear in the Preview Screen.
+
+![AddAppBar.avif](imgs/AddAppBar.avif)
+
+## 6. Multi-Language Preview
+
+If you've enabled the multi-language feature for your project, you can use this to preview your app in different languages. Just select the language from the dropdown menu, and the text in your app will be translated accordingly.
+
+:::tip
+This feature is valuable for testing your app across multiple locales without needing to run your app.
+:::
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'
+}}>
+    <iframe 
+        src="https://demo.arcade.software/E6otMpbcKewMYCfkjl9d?embed&show_copy_link=true"
+        title="Sharing a Project with a User"
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+
+## 7. Dark/Light Mode
+
+This toggle lets you switch between light and dark mode in your app, allowing you to ensure your design looks great in both modes. This feature is only available if you've enabled dark mode support in your project.
+
+## 8. Display Device
+
+You can use this to show the device frame in the preview. This might come in handy if you want to visualize better or capture the user interface on a particular device.
```

Comment:
Add a small note that this is useful when checking your screen with safe area or how it will look with iPhone or Android notches etc (you can frame it better)

Reply:
Done!